README.md

```diff
@@ -55,9 +82,13 @@ Extend the theme by creating a file `_layouts/website/page.html` in your book wi
 {% endblock %}
 ```
 
-##### Add short description for the home page
```

Comment:
The fact that the README is used on the writer's profile home page is incidental. For people who host their books on GitHub, the README is displayed on the project's landing page. So there's no need to mention it. I'll just simplify this paragraph when merging.